modules/virt-creating-filesystem-fusion-access-san.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * virt/fusion_access_SAN/install-configure-fusion-access-san.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-filesystem-fusion-access-san_{context}"]
+= Creating a file system with {FusionSAN}
+
+You need to create a file system to represent your required storage.
+
+The file system is based on the storage available in the worker nodes you selected when creating the storage cluster.
+
+.Prerequisites
+
+* You created a {FusionSAN} storage cluster.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Storage* -> *{FusionSAN}*.
+
+. In the *File systems* tab, click *Create file system*.
+
+. Enter a *Name* for the new file system.
+
+. Select the LUNs that you want to use as the storage volumes for your file system.
+
+. Click *Create file system*. 
++
+The *{FusionSAN}* page reloads, and the new file system appears in the *File systems* tab.
+
+Repeat this procedure for each file system that you want to create.
```

Comment:
This line is currently outside of the procedure and after conversion to DITA, will not be carried over. It needs to either be appended to a previous step, but I would probably mark it as a next step.
```suggestion
.Next steps

Repeat this procedure for each file system that you want to create.
```